src/components/BooksList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+function BooksList({ books }) {
+  const bookList = books.map(book => (
+    <Book
+      id={book.id}
+      title={book.title}
+      category={book.category}
+      key={book.id}
+    />
```

Comment:
- pass the book as a whole object instead of splitting it's properties, i think that will be better. 
something like this : 
```
  <Book key={index} book={book} /> 
```